style.css

```diff
@@ -0,0 +1,642 @@
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+/* Navigation Bar */
+nav {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 45px;
+  background-color: white;
+  top: 0;
+  border-bottom: 1px solid #e2e2e2;
+}
+
+/* Navigation Bar Left */
+.nav-left {
+  display: flex;
+  flex-direction: row;
+}
+
+aside a {
+  font-size: 14px;
+  margin-left: 30px;
+  margin-top: 5px;
+}
+
+.nav-left a {
+  margin: 15px;
+  font-family: helvetica, arial, sans-serif;
+  font-weight: 700;
+}
+
+.nav-left i {
+  margin: 15px;
+}
+
+main section p {
+  margin-bottom: 0.78125rem;
+  margin-top: 0;
+  font-family: nyt-imperial, georgia, 'times new roman', times, serif;
+  font-size: 1.3rem;
+  line-height: 1.5625rem;
+}
+
+/* Navigation Bar Center */
+.nav-center {
+  display: inline-flex;
+  align-items: center;
+}
+
+aside img {
+  height: 80px;
+  width: 90px;
+  margin: 5px;
+}
+
+.nav-center img {
+  height: 45px;
+}
+
+/* Navigation Bar Right */
+.nav-right button {
+  margin: 15px;
+  background-color: rgb(86, 123, 149);
+  border: solid;
+  border-color: rgb(86, 123, 149);
+}
+
+.nav-button {
+  color: white;
+  font-weight: 700;
+  border: transparent;
+  padding: 7px;
+  border-radius: 5px;
+}
+
+/* Main Article */
+main {
+  width: 100%;
+}
+
+.main-article {
+  width: 90%;
+  margin-left: 10%;
+}
+
+figcaption {
+  font-family: nyt-imperial, georgia, 'times new roman', times, serif;
+  font-size: 15px;
+  color: #666;
+  height: 20%;
+  width: 75%;
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.main-article h2 {
+  font-family: nyt-cheltenham, georgia, 'times new roman', times, serif;
+  font-weight: 700;
+  font-style: italic;
+  line-height: 3rem;
+  font-size: 2.2rem;
+  margin-top: 3em;
+  margin-bottom: 0.8em;
+  width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#main-img1 {
+  width: 90%;
+  height: 90%;
+}
+
+.main-article p {
+  padding-right: 35%;
+}
+
+/* Main Section */
+main section {
+  width: 40%;
+  margin-left: 30%;
+}
+
+.main-section-icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.main-section-icons p {
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 500;
+  font-size: 13px;
+  color: #333;
+}
+
+.icon-alignment i {
+  margin-left: 10px;
+}
+
+#main-section-img2 {
+  height: 70%;
+  width: 120%;
+  margin-left: -10%;
+  margin-right: 0;
+}
+
+.main-section-img2-span {
+  height: 70%;
+  width: 120%;
+  margin-left: -10%;
+  margin-right: 0;
+}
+
+#main-section-belowicons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+#main-section-belowicons button {
+  background-color: rgb(86, 123, 149);
+  border: 1px solid rgb(50, 104, 145);
+  color: rgb(255, 255, 255);
+  padding: 11px 12px 8px;
+  width: 400px;
+  border-radius: 3px;
+}
+
+#section-align {
+  margin-bottom: -100px;
+}
```

Comment:
- [ ] Use CSS `classes` instead of `id's`

Reply:
Thank you for the suggestions, we made the changes